flink-core/src/test/java/org/apache/flink/api/common/io/DelimitedInputFormatTest.java

```diff
@@ -431,6 +431,34 @@ public void testDelimiterOnBufferBoundary() throws IOException {
 		format.close();
 	}
 
+	@Test
+	public void testDelimiterOnBufferBoundaryWithWholeFileSplit() throws IOException {
```

Comment:
Maybe we can deduplcate common codes with previous `testDelimiterOnBufferBoundary`, because only one sentence is different. :)